.github/workflows/contracts.yml

```
@@ -13,16 +13,16 @@ on:
+      - "archive/legacy-evm/**"
       - "soroban/**"
       - "package.json"
       - "package-lock.json"
       - ".github/workflows/contracts.yml"
 
 jobs:
   solidity:
-    name: Solidity Prototype Checks
-    if: ${{ hashFiles('contracts/**/*.sol') != '' }}
+    name: Archived Solidity Prototype Checks
+    if: ${{ hashFiles('archive/legacy-evm/contracts/**/*.sol') != '' }}
```

actionlint (1.7.12)

[error] calling function "hashFiles" is not allowed here. "hashFiles" is only available in "jobs.<job_id>.steps.continue-on-error", "jobs.<job_id>.steps.env", "jobs.<job_id>.steps.if", "jobs.<job_id>.steps.name", "jobs.<job_id>.steps.run", "jobs.<job_id>.steps.timeout-minutes", "jobs.<job_id>.steps.with", "jobs.<job_id>.steps.working-directory". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)